misk-redis/src/testFixtures/kotlin/misk/redis/testing/RealRedisTestModule.kt

```diff
@@ -27,6 +27,7 @@ class RealRedisTestModule(
         RedisModule(redisReplicationGroupConfig, connectionPoolConfig, useSsl)
       ).with(TestUnifiedJedisModule(connectionPoolConfig, useSsl))
     )
+    install(RedisTestFlushModule())
```

Comment:
This is being manually installed, whereas it's almost always needed when real Redis is used in tests, so installingit here makes more sense.